dbt_platform_helper/commands/copilot.py

```diff
@@ -313,6 +316,8 @@ def make_addons():
         log_destination_arns = get_log_destination_arn()
 
         if addon_type in ["s3", "s3-policy"]:
+            if config[addon_name].get("serve_static"):
```

Comment:
We don't need svc or env addons, for a static bucket because we don't need the svc addon policy. I'm not adding logic for digital workspace because we're migrating them off copilot and they don't require this functionality, so I think it's wasted effort